fdbserver/DataDistribution.actor.cpp

```diff
@@ -742,6 +742,22 @@ ACTOR Future<Void> monitorPhysicalShardStatus(Database cx, Reference<PhysicalSha
 	}
 }
 
+ACTOR Future<Void> dataDistributionCore(Reference<DataDistributionRuntimeMonitor> self) {
```

Comment:
When do we need this ACTOR, if we really need some asynchronous processing for `DataDistributionRuntimeMonitor`, can we add those as a member Future<>, let's use `DataDistributionRuntimeMonitor` class as the single interface.